File: bindings/php/src/Exception/EvaluateException.php
<?php

declare(strict_types=1);

/* SPDX-License-Identifier: Apache-2.0 */

namespace Goplasmatic\Datalogic\Exception;

/** Thrown when rule evaluation fails (Thrown, NaN, runtime, …). */
final class EvaluateException extends DatalogicException {}

File: bindings/jvm/src/main/java/com/goplasmatic/datalogic/EvaluateException.java
/* SPDX-License-Identifier: Apache-2.0 */
package com.goplasmatic.datalogic;

/** Thrown when rule evaluation fails (Thrown, NaN, runtime, …). */
public final class EvaluateException extends DatalogicException {
    EvaluateException(String message, String errorType, String operatorName, String pathJson) {
        super(message, errorType, operatorName, pathJson);
    }
}
